tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -37,6 +37,7 @@
 from tensorflow.python.ops import variable_scope as vs
 from tensorflow.python.platform import tf_logging as logging
 from tensorflow.python.util import nest
+from tensorflow.python.ops.math_ops import tanh
```

Comment:
please import math_ops and use math_ops.tanh.  also move the import up so it's sorted in lexicographic order.